simulation_parameters/microbe_stage/biomes.json

```diff
@@ -1861,7 +1861,7 @@
           "Radius": 1,
           "ChunkScale": 1,
           "PhysicsDensity": 4000,
-          "Size": 1000,
+          "Size": 3,
```

Comment:
Why did you change the size? I think the size is 1000 here to prevent this being accidentally engulfed (if the engulfed property is accidentally enabled on this).

Reply:
And I'll note that this "size" is really the engulfing size for engulfing calculations and impacts nothing else.

Reply:
Ah, okay. I thought the engulfing was intentional. It damages you, but then quickly dissolves. I can change it.

Reply:
The size was 1000 before as it was for gameplay purposes to prevent eating the ice. It could be changed but I'd want someone from the game design team to weigh-in on such a change (I personally think that preventing the player from eating the ice for no other purpose than to receive damage is better than allowing engulfing the ice).